modules/lab/src/main/java/org/locationtech/jtslab/simplify/RadialDistanceByAngleSimplifier.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (c) 2016 Vivid Solutions.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+package org.locationtech.jtslab.simplify;
+
+import org.locationtech.jts.algorithm.Angle;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.CoordinateSequence;
+import org.locationtech.jts.geom.CoordinateSequenceFactory;
+import org.locationtech.jts.geom.CoordinateSequences;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.geom.LineString;
+import org.locationtech.jts.geom.Lineal;
+
+
+    /**
+     * This class simplifies a line string by removing points that create a sharp turn within a radius from previous point.
+     * This class follows the JTS simplifier pattern
+     *
+     * @author 
+     *   Harel M (https://github.com/harelm)
```

Comment:
@FObermaier can you say more about the provenance here?  If you can link to the project, that might help.  Generally, if we get a quick 'ok' from the original author for creating the derivative work, that'd be best.